scout/load/setup.py

```diff
@@ -119,6 +120,10 @@ def setup_scout(
     else:
         exac_lines = fetch_exac_constraint()
 
+    # Load cytobands into cytoband collection
+    for genome_build, cytobands_path, in cytoband_files.items():
```

Comment:
I find trailing commas scary. Just saying. 😆

Reply:
Me too! This was a typo, good that you noticed!